shell/platform/darwin/ios/framework/Source/FlutterView.mm

```diff
@@ -64,11 +64,11 @@ - (void)layoutSubviews {
 }
 
 + (Class)layerClass {
-#if TARGET_IPHONE_SIMULATOR
-  return [CALayer class];
-#else   // TARGET_IPHONE_SIMULATOR
+// #if TARGET_IPHONE_SIMULATOR
```

Comment:
Please don't check-in commented code. If there are annotations needed to explain the context of code, please add some text comments.